polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -101,11 +105,11 @@
   border-top-right-radius: var(--p-border-radius-2);
 
   #{$se23} & {
-    border-top-left-radius: var(--p-border-radius-3);
-    border-top-right-radius: var(--p-border-radius-3);
+    border-radius: 0;
```

Comment:
These are for xs screens to fix a bug where the subdued sections had the wrong border radius